emission/analysis/intake/segmentation/trip_segmentation_methods/dwell_segmentation_dist_filter.py

```diff
@@ -144,7 +144,7 @@ def segment_into_trips(self, timeseries, time_query):
                 logging.debug("Found %d transitions after last point, not ending trip..." % len(stopped_moving_after_last))
         return segmentation_points
 
-    def has_trip_ended(self, lastPoint, currPoint, timeseries):
+    def has_trip_ended(self, lastPoint, currPoint, timeseries, motion_df):
```

Comment:
if you are already passing in the `motion_df` here, why do you need to also pass in the timeseries?

Reply:
it's being used here :

       timeseries.invalidate_raw_entry(currPoint["_id"])